src/test/java/uk/gov/companieshouse/presenter/account/controller/PresenterAccountControllerTest.java

```diff
@@ -54,12 +57,12 @@ void testHealthCheckEndpointSuccessResponse() {
     @DisplayName("Return 201 when valid presenter details is submitted")
     void testCreatePresenterAccountSuccessResponse() {
         final String id = "id";
-        when(presenterAccountService.createPresenterAccount(presenterRequest)).thenReturn(id);
+        when(presenterAccountService.createPresenterAccount(presenterAccountDetailsRequest)).thenReturn(id);
 
-        var response = presenterAccountController.createPresenterAccount(presenterRequest);
+        var response = presenterAccountController.createPresenterAccount(presenterAccountDetailsRequest);
         var header = response.getHeaders().getFirst("Location");
         assertThat(response.getStatusCode(), is(HttpStatus.CREATED));
-        assertTrue(header.contains(PRESENTER_ACCOUNT + id));
+        Assertions.assertTrue(header != null && header.contains(PRESENTER_ACCOUNT + id));
```

Comment:
Did you mean to change this from the static function?

Reply:
It was a warning from the IDE suggesting this was better.
I used the IDE warnings to find issues so I try to resolve any that are within the code that I edit.
I'm not sure how I only managed to change one of the functions.

Anyway, I think that static function works fine so I'll revert and disable the warning in my IDE.